server/agents/source-agents/open-weather-map/interfaces.ts

```diff
@@ -17,20 +18,22 @@ export interface Configuration {
     name: string;
     targetLocation?: string;
     jobId?: string;
+    jobOwner: string;
 }
 
 export interface AgentResult {
     agentId: string;
     jobId: string;
+    jobOwner: string;
```

Comment:
Is the "jobId" already a pointer to the owner and we can get rid of the owner in the agents?